### ISSUE TITLE
Datastore não processa inserção de novas linhas em csv de recurso atualizado

### ISSUE THREAD

**Issue by Andrelamor** [dados-mg/issues#58]
## Descrição do problema:

Subi 2 novas linhas no dataset de 'doações selo amigo do estado', mas elas não aparecem no preview do datastore:

Embora o arquivo baixado mostra que ele agora tem 47 linhas, o preview só dá opção de mostrar 44 linhas.

![image](https://user-images.githubusercontent.com/52294411/129925778-c6a51493-0de6-4a6c-809a-90ed6cef9c59.png)

No painel de gestão do recurso, o erro informado:

![image](https://user-images.githubusercontent.com/52294411/129925590-fac927c6-fecf-45a2-9d9d-8f61f8ba9f4f.png)

**Comment by fjuniorr** [comment]
@Andrelamor movi esse issue do repo `dados-mg/dpckan` para esse aqui (`dados-mg/issues`) tendo em vista que o problema não está relacionado ao `dpckan`.

Vou pedir pra DTI avaliar se o log `/var/log/apache2/datapusher.error.log` tem alguma informação relevante.

**Comment by Andrelamor** [comment]
claro, até pq não usei o pacote pra atualizar o recurso, mas sim a interface gráfica

**Comment by Andrelamor** [comment]
@gabrielbdornas @fjuniorr:

1. are we running the DataPusher on the same host as CKAN?

2. is our `site_url` setting in your CKAN INI file correct? 

3. source: https://stackoverflow.com/questions/51544342/error-uploading-excel-file-to-ckan-datastore

**Comment by gabrielbdornas** [comment]
> @Andrelamor movi esse issue do repo `dados-mg/dpckan` para esse aqui (`dados-mg/issues`) tendo em vista que o problema não está relacionado ao `dpckan`.
> 
> Vou pedir pra DTI avaliar se o log `/var/log/apache2/datapusher.error.log` tem alguma informação relevante.

@fjuniorr, @Andrelamor e eu queríamos saber se este chamado na TI foi aberto. Recentemente o problema está aparecendo e não estamos conseguindo atualizar o datapusher nem via aplicação. Isso tem ocorrido tanto em produção quanto em homologação. 

Maristela e eu quando criamos os conjuntos da FHEMIG em produção tivemos o seguinte erro `Erro no envio: An Error occurred while sending the job: 500 Server Error: Internal Server Error for url: http://0.0.0.0:8800/job`:

![WhatsApp Image 2022-04-19 at 4 01 31 PM](https://user-images.githubusercontent.com/49699290/165635648-52c20939-db39-4d10-993d-32b8974caa3a.jpeg)

Abaixo outro exemplo do conjunto da vale em homologação. Detalhe sobre a atualização via `dpckan` é que a versão atualizada do arquivo está disponível via url ou botão "baixar".

![image](https://user-images.githubusercontent.com/49699290/165635891-71f88738-ebef-4795-bb41-3d288f8faab9.png)

**Comment by fjuniorr** [comment]
> @fjuniorr, @Andrelamor e eu queríamos saber se este chamado na TI foi aberto.

Não abri não. Relata por favor no https://atendimentoti.cge.mg.gov.br/.

A mensagem de _Could not connect to DataPusher_ parece ser nova né? As vezes na atualização para o CKAN 2.9 algo aconteceu.
